src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -36,7 +36,6 @@ export const NavigationTabs: React.FC<NavigationTabsProps> = ({ partner }) => {
       {
         name: "Artists",
         href: route("/artists"),
-        exact: true,
```

Comment:
I found that [RouteTab](https://github.com/artsy/force/blob/cb02c6bc0fb53007ab2f0a8e3b55f06d4b1420e8/src/v2/Components/RouteTabs.tsx#L13) doesn't use `exact` field to define the active tab. I will fix it in a separate branch.